### ISSUE TITLE
-P, --process-only to display process name without the PID in brackets

### ISSUE THREAD

**Issue by porg** [martinpitt/fatrace#14]
## Feature request
- When you use `--process-only` or `-P` `fatrace` outputs only the process name

## Advantages & User Goals
- Shorter more readable output, when one is mostly interested in "which process" (name) and not so much "which instance" (PID).
- This is especially useful when logging in multiple sessions (where the PID) may change, but in an accumulated output want to cluster by process-name.

**Comment by martinpitt** [comment]
I find this a bit too specialized TBH. Proposal #15 is more generic, and would avoid introducing lots of special-cased options for similar output tuning.

**Comment by porg** [comment]
I agree! If you consider #15 feel free to close down this here as unnecessary complicated redundant.
